rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/identify/IdentifiedArchives.java

```diff
@@ -53,8 +53,7 @@ public static void addMappingsFrom(File file)
                     continue;
                 String[] parts = StringUtils.split(line, ' ');
                 if (parts.length < 2)
-                    throw new IllegalArgumentException("Expected 'SHA1 GROUP_ID:ARTIFACT_ID:VERSION[:COORDINATE]', but was [" + line + "] in ["
-                                + file + "]");
+                    throw new IllegalArgumentException("Expected 'SHA1 GROUP_ID:ARTIFACT_ID:[PACKAGING:[COORDINATE:]]VERSION', but was: \n" + line + "\n\tin [" + file + "]");
```

Comment:
The reason is that it is mixing two very long strings, a path and the line. Which forces the user to scroll horizontally, which is really very unpleasant. Breaking does not help as it creates an ugly blob of text which is hard to select by mouse and copy. Personally, I would put paths on new line every time.